docs/config.md

```diff
@@ -132,6 +132,9 @@ For a good example, see https://develop.element.io/config.json.
 1. `audioStreamUrl`: If supplied, show an option on Jitsi widgets to stream
    audio using Jitsi's live streaming feature. This option is experimental and
    may be removed at any time without notice.
+1. `voipObeyAssertedIdentity`: If set, MSC3086 asserted identity messages sent
```

Comment:
Would a ‘voip’ object be better for the future? Just an idea.

Reply:
yep, good point